src/cli.js

```diff
@@ -0,0 +1,132 @@
+#!/usr/bin/env node
+'use strict';
+
+var _ = require('underscore'),
+    Q = require('q'),
+    AWS = require('aws-sdk'),
+    minimist = require('minimist'),
+    Synchronizer = require('./Synchronizer'),
+    startupPromise = Q.when(),
+    argvOpts, argv, argsFailed, options;
+
+argvOpts = {
+   string: [ 'master', 'slaves', 'ignore-atts', 'starting-key', 'profile', 'role-arn', 'mfa-serial', 'mfa-token' ],
+   'boolean': [ 'write-missing', 'write-differing', 'delete-extra' ],
+   'default': {
+      'write-missing': false,
+      'write-differing': false,
+      'delete-extra': false,
+   },
+   alias: {
+      master: 'm',
+      slaves: [ 's', 'slave' ],
+      'ignore-atts': [ 'ignore', 'ignore-att' ],
+   },
+};
+
+argv = minimist(process.argv.slice(2), argvOpts);
+
+function mapTableName(type, name) {
+   var parts = name.split(':');
+
+   if (parts.length !== 2) {
+      console.log('Your table name must be supplied in two parts: "<region>:<table-name>"');
+      console.log('This %s table does not meet that requirement:', type, name);
+      argsFailed = true;
```

Comment:
I understand why this is here in the function, but I'm a tad wary of doing so. It gives `mapTableName` two jobs, validate/parse the table name and update the script's validation state. In the long run, this is probably fine. However, just want to be sure.